modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -493,6 +493,27 @@ $ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
 CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ----
 
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
+== Running in single node cluster
+
+Running tests on a single node cluster imposes limitations:
+
+* Longer timeouts for certain tests, including SRI-OV and SCTP tests
+* Tests requiring master/worker nodes are skipped
+
+Longer timeouts concern SRI-OV and SCTP tests. Reconfiguration requiring node reboots causes a reboot of the entire environment, including the OpenShift control plane, and therefore takes longer to complete. All PTP tests requiring a master and worker node are skipped. No additional configuration is needed because the tests check for the number of nodes at startup and adjust test behavior accordingly.
+
+PTP tests can run in Discovery mode. The tests will look for a PTP master configured outside of the cluster.
+
+To enable Discovery mode, the tests must be instructed by setting the `DISCOVERY_MODE` environment variable as follows:
+
+[source,terminal]
+----
+$ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
+DISCOVERY_MODE=true registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
+----
+
```

Comment:
One think we forgot to mention is that we need to specify `ROLE_WORKER_CNF=master` in case of SNO because that's the only available role.
cc @mmirecki is that correct? Also, I think it makes sense to add this note to u/s docs too.

Reply:
Not 100% sure now, but that seems logical as we only have a master node